### DISCUSSION TITLE
Add Dialog option

### DISCUSSION THREAD

**Discussion by Charvelx04** [SubtitleEdit/subtitleedit#6055] [Ideas]
Is there an option, or can there be, an option to create a dialog?

For example:

This isn't what I had in mind.
Oh, really? This isn't it?!

Then, when you highlight the line.. click on, say, Edit, or maybe a right click option - "Add Dialog" so we get the following:

- This isn't what I had in mind.
- Oh, really? This isn't it?!

I know there's an issue where OCR doesn't fix this:

This isn't what I had in mind.
- Oh, really? This isn't it?!

Or when some subs have this:
This isn't what I had in mind. - Oh, really? This isn't it?

But those are easy to spot and fix.
![Untitled](https://user-images.githubusercontent.com/35577096/175787020-9e746a0c-a826-41a3-8b21-456afa6d2be8.jpg)

**Comment by OmrSi** [comment]
@Charvelx04 There is a shortcut for this called `Toggle dialog dashes`.

**Comment by Charvelx04** [comment]
Thank you.
